docs/local-setup/local-dev-testnet.md

```diff
@@ -84,7 +84,7 @@ near create_account <yourcontractname> --masterAccount=<login> --initialBalance
 ```
 
 #### masterAccount : specifies which account's key should be used to deploy the contract. This is the account ID you specified when you created your wallet. 
-#### initialBalance : specifies how much to seed the contract balance with. The min is 10 NEAR. However, in this case the amount is measured in attoNEAR (1 NEAR is 10^18 attoNEARs). Thus, you want to put at least 1000000000000000000 attoNEAR.
+#### initialBalance : specifies how much to seed the contract balance with. The min is 10 NEAR.
```

Comment:
I believe that "10 NEAR" is not the min value, but rather a default one. @janedegtiareva Is my understanding correct?